src/components/VaultActions/index.tsx

```diff
@@ -72,15 +72,13 @@ function VaultActionsDesktop({ vault, hasSupplyPosition }: { hasSupplyPosition:
         </div>
       )}
 
-      <div className="rounded-[12px] bg-button-supply-muted">
```

Comment:
Undo this, similarly this is needed to provide background color like the earn ones:
![Screenshot 2025-05-09 at 11 46 52](https://github.com/user-attachments/assets/a38178d5-0fc7-463b-b850-395870dce377)